### ISSUE TITLE
sequenceserver failed to recognize FASTA files

### ISSUE THREAD

**Issue by mictadlo** [wurmlab/sequenceserver#410]
Hi,
I am using sequenceserver 1.0.11 inside this [docker-container](`https://github.com/fikipollo/blast-docker`). The below genomes were not recognise by sequenceserver probably due by having many `N`s at the beginning:

- ftp://ftp.solgenomics.net/genomes/Nicotiana_tabacum/edwards_et_al_2017/assembly/Nitab-v4.5_genome_Chr_Edwards2017.fasta.gz
- http://hgdownload.soe.ucsc.edu/goldenPath/hg38/bigZips/hg38.fa.gz

How is it possible to get them inside sequenceserver?

Thank you in advance,

Michal

**Comment by yannickwurm** [comment]
Hi Michal,
thanks for the report. I think this means we: 
 - simply need to skip Ns  (i.e. take more sequence if too many Ns at the beginning)
 - or if uncertain, blindly trust the user.

**Comment by yannickwurm** [comment]
Logic is here - and actually does appear to skip N's so Im unsure where it is going wrong?

https://github.com/wurmlab/sequenceserver/blob/dbdc4eb6f43f9673a3a9a016cd9761e3303901d2/public/js/search.js

**Comment by yeban** [comment]
@yannickwurm I think this about importing them as databases.

@mictadlo Thanks for reporting this. For now, it seems you will need to run makeblastdb on the FASTA files yourself before running SequenceServer docker image.

```
makeblastdb -parse_seqids -hash_index -in hg38.fa -dbtype nucl -title 'hg38'
```

**Comment by yeban** [comment]
Thanks again for reporting this issue. The issue is now fixed (a31b7d3) and will be a part of the 2.0 release.

**Comment by yeban** [comment]
For the record, I was able to reproduce this only for the Nitab genome and not for hg38.
